nilearn/_utils/ndimage.py

```diff
@@ -6,12 +6,12 @@
 
 import numpy as np
 from scipy import ndimage
-
-
```

Comment:
Ideally, it would be better to leave other things not to change which are not related to your PR. Sometimes reviewing PR might get complex. Just a suggestion. :)